README.md

```diff
@@ -571,6 +571,7 @@ Below is the complete list of parameters that can be set using environment varia
 - **REDMINE_RELATIVE_URL_ROOT**: The relative url of the Redmine server, e.g. `/redmine`. No default.
 - **REDMINE_ATTACHMENTS_DIR**: The attachments directory. Defaults to `/home/redmine/data/files`
 - **REDMINE_SECRET_TOKEN**: Secret key for verifying cookie session data integrity. Defaults to a random alphanumeric string.
+- **REDMINE_RMAGICK_FONT_PATH**: The rmagick_font_path for the png export function of GANTT to work. No default.
```

Comment:
I think saying it has no default is a bit misleading.  I think it will lead people to think that they have to set the value which is not the case.  I'd prefer Defaults to ``

Also is GANTT a plugin, if so it would be more clear if stated as the name of the plugin (Gantt plugin?), as I'm pretty sure redmine doesn't have built-in GANTT support.

Note: REDMINE_RELATIVE_URL_ROOT which states it has no default, actually has a default of '/'. I'll add a commit to fix this.

Reply:
This person says they set their path to `/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf` which does exist in our container.  Would this be an acceptable default?
https://github.com/docker-library/redmine/issues/65#issuecomment-297280735

Reply:
> as I'm pretty sure redmine doesn't have built-in GANTT support.
Nvm I guess there is a built-in gantt chart. 
http://www.redmine.org/projects/redmine/wiki/RedmineGantt

In that case we should definately set the REDMINE_RMAGICK_FONT_PATH to default to something that makes this work.